Formula/fb-client.rb

```diff
@@ -44,16 +44,17 @@ def install
     # avoid pycurl error about compile-time and link-time curl version mismatch
     ENV.delete "SDKROOT"
 
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor"/Language::Python.site_packages("python3")
+    python3 = "python3.10"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor"/Language::Python.site_packages(python3)
 
     # avoid error about libcurl link-time and compile-time ssl backend mismatch
     resource("pycurl").stage do
-      system "python3", *Language::Python.setup_install_args(libexec/"vendor"),
+      system python3, *Language::Python.setup_install_args(libexec/"vendor"),
                         "--curl-config=#{Formula["curl"].opt_bin}/curl-config"
```

Comment:
```suggestion
      system python3, *Language::Python.setup_install_args(libexec/"vendor", python3),
                      "--curl-config=#{Formula["curl"].opt_bin}/curl-config",
                      "--install-data=#{prefix}"
```
Let's always make sure `setup_install_args` gets a second argument (or else it may pass a wrong `--install-lib`).

Also, passing `--instal-data` should fix your build error, I think. (This is due to changes in Python3).

Reply:
I saw "error: could not create '/opt/homebrew/share/doc/pycurl': Operation not permitted" on the ci log.
The build works fine on Ubuntu22.04 (x86_64). Is install-data a tweak to the arm device?

Reply:
It's a tweak to where it should install things that go into `share`. No idea why it failed only on ARM.

Reply:
Thanks for answering.